src/main/java/com/thebois/models/world/structures/House.java

```diff
@@ -30,14 +30,13 @@ public House(final Position position) {
         super(position, StructureType.HOUSE, generateNeededItemsInventory());
     }
 
-    private static IInventory generateNeededItemsInventory() {
-        final IInventory neededItems = new Inventory();
+    private static Map<ItemType, Integer> generateNeededItemsInventory() {
```

Comment:
style

You're no longer generating an inventory, so maybe just something like `getNeededItems`.

Reply:
Forgot to change it before I pushed it up, but I changed it to `getNeededItemsMap`